### DISCUSSION TITLE
API documents and instructions

### DISCUSSION THREAD

**Discussion by wswapneel** [GreaterWMS/GreaterWMS#280] [General]
Hi,

I am so thankful for GreatWMS, a very good project. I deployed the development server yesterday but can't find the API documents and instructions to create and connect to another order management and courier services like FedEx, UPS... I would like to know if you have the same kind of set of instructions to create API's as for deployment of the server. Your help is greatly appreciated,  Thank you !  Neel

**Comment by Singosgu** [comment]
new version 2.1.37 has contain the api docs

url is /docs/
